src/Microsoft.Health.Dicom.Blob/Utilities/ExternalBlobDataStoreConfiguration.cs

```diff
@@ -7,7 +7,7 @@
 
 namespace Microsoft.Health.Dicom.Blob.Utilities;
 
-internal class ExternalBlobDataStoreConfiguration
+public class ExternalBlobDataStoreConfiguration
```

Comment:
Why the change? If it's for testing, you can add an `InternalsVisibleToAttribute` like https://github.com/microsoft/dicom-server/blob/3c88dbad62d6647661fabf611e0f186d7419411c/src/Microsoft.Health.Dicom.Blob/Properties/AssemblyInfo.cs#L10